packages/react/src/Dialog/Dialog.tsx

```diff
@@ -476,6 +469,13 @@ const _Dialog = React.forwardRef<HTMLDivElement, React.PropsWithChildren<DialogP
           }),
         )
 
+  useFocusTrap({
```

Comment:
(non blocking) Do we still need to move this below?

Reply:
Nope! We can move this back.